test/signed-log.spec.js

```diff
@@ -13,27 +13,23 @@ const dataDir = './ipfs/tests/log'
 
 const repoConf = {
   storageBackends: {
-    blocks: DatastoreLevel,
-  },
+    blocks: DatastoreLevel
+  }
 }
 
-const ipfsConf = { 
+const ipfsConf = {
   repo: new IPFSRepo(dataDir, repoConf),
   EXPERIMENTAL: {
     pubsub: true,
     dht: false,
-    sharding: false,
-  },
+    sharding: false
+  }
 }
 
 let ipfs, key1, key2, key3
 
-const last = (arr) => {
```

Comment:
Was this not used?

Reply:
Can't find any occurrence of it being used at that file. I think it's a copy and past leftover

Reply:
Cool. In that case, this looks pretty good to me, then.